### ISSUE TITLE
Scan Series has different results than Scan Library and Breaks Metadata

### ISSUE THREAD

**Issue by armouredking** [Kareadita/Kavita#1244]
Brave V1.38.111 Chromium 101.0.4951.54
Windows 11 Client, Docker CE Server using Latest tag image 0.5.2

**Describe the bug**
Scan Series results in different outcomes than Scan Library. This tends to happen whenever there are specials in a series. "Scan Library" will find specials and present them without issue. Now lets say you're doing some edits to fix covers on individual chapters so you hit "Scan Series" as that's the recommended action vice "Refresh Covers" and boom - all the Specials vanish. Performing another "Scan Library" will cause them to show back up, but any edited metadata (such as custom covers) is gone - from _both_ the actual volumes / chapters and the specials.

The odd thing about the covers in particular is that sometimes they still show up in the Details section of the metadata viewer, but attempting to resave them will cause them to revert. Guessing this may be a caching issue; attempted with InPrivate as well to verify not browser cache related.

It should be noted that no actual edits are required to reproduce this; you can cause this simply by adding a new series, adding the files for it, hitting Scan Library, notice they show up, hit Scan Series, watch the specials disappear.

Layout is per guide docs as this is a new setup. Using _cover to specify images for use in the CBZs is not lost, only those made with the metadata viewer.

**Expected behavior**
You don't lose changes made previously by doing updates and / or Scan Series and Scan Library do not produce different results.

**Comment by majora2007** [comment]
Hi, Can you give me a very small snippet of your naming convention and folder layout for this series so I can try to reproduce? I don't suffer from this issue myself. 

Also, as a note, anytime you do refresh cover will override and reset all covers. It forces a refresh and ignores if you manually reset. Hence the warning and why you almost never need to use it. Scan series still calls the cover refreshing, but doesn't force anything.

**Comment by armouredking** [comment]
I gave up trying to get it to work, so current setup is without specials, but when they had specials it looked like this:

Root-
 - Yomeiro Choice
   - Specials
     - SP01 Omake Fuck to the Future.cbz
     - ...
     - SP06 Omake Oh no... Tiny!.cbz
   - Yomeiro Choice V1 - C01.cbz
   - ...

**Comment by majora2007** [comment]
Hmm interesting, let me try to work it out. I've never put the Special marker at the beginning of the file, so maybe that has something to do. I'll get back to you.

**Comment by armouredking** [comment]
I'm not sure if the cover thing is related to this particular issue or not, but it seems to have come back as well despite having no specials. These were working last night (ie the volume covers showed the uploaded images), and per the metadata system they are still there, but the images are now wrong. When I got rid of the specials I also removed the files, scanned, and put them back after renaming them sans specials (basically a Plex dance) for a final scan to try and clear any caches that might have been created.

![image](https://user-images.githubusercontent.com/16038428/167307398-8c776ed2-7344-4f96-92ba-aef0fc5e9a0f.png)
![image](https://user-images.githubusercontent.com/16038428/167307415-faa7cfc0-393e-4448-8947-a48c162eb399.png)

Logging in from another browser (ie using Firefox instead so there is no browser cache) does not fix them either.
